### DISCUSSION TITLE
Welcome to Keep Discussions Board!

### DISCUSSION THREAD

**Discussion by talboren** [keephq/keep#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by guileh** [comment]
Hey @talboren! What a wonderful project!

**Reply by talboren** [reply]
Thanks @guileh! are you planning on trying Keep? Welcome to join our [Slack community](https://slack.keephq.dev)

**Reply by guileh** [reply]
Today, I came across Keep while searching for alert management solutions. As part of a team that provides a monitoring platform, Keep appears to be a perfect match for a problem we are currently addressing. We will conduct a trial with an internal application to learn more about its features.

**Reply by talboren** [reply]
Gotcha! Would love to learn more about the use case, and anyway, here for any question :)
